crates/bevy_render/Cargo.toml

```diff
@@ -73,6 +73,7 @@ parking_lot = "0.12.1"
 regex = "1.5"
 ddsfile = { version = "0.5.0", optional = true }
 ktx2 = { version = "0.3.0", optional = true }
+naga_oil = { git="https://github.com/robtfm/naga_oil", branch="bevy-changes" }
```

Comment:
We should start discussing the plan for pulling naga_oil under the bevy umbrella. I'm thinking:

1. We create a new empty `bevyengine/naga_oil` repo.
2. You create a new pull request adding the whole contents of naga_oil. We give you merge rights on the naga_oil repo (as you are certainly qualified to maintain it). We'd also add the current bevy maintainers. It would still use the normal bevy dev process. Controversial changes (ex: things that meaningfully affect syntax or developer workflows) would need to go through the normal approval process.
3. You keep crates.io publishing rights, but also add me.
4. We merge the pr (after review) and publish a new version.
5. We update this PR and merge it.

Reply:
this sounds fine to me. i can also just transfer the repo if you prefer.

Reply:
> i can also just transfer the repo if you prefer.

Ah yeah thats a good call. Lets do that to preserve history and the like